licenses-report.txt

```diff
@@ -324,28 +744,44 @@ The following software have components provided under the terms of this license:
 
 - Guava: Google Core Libraries for Java (from https://github.com/google/guava.git)
 - HdrHistogram (from http://hdrhistogram.github.io/HdrHistogram/)
+- Joda-Time (from http://www.joda.org/joda-time/)
+- Microsoft Azure client library for Blob Storage (from https://github.com/Azure/azure-sdk-for-java)
+- Prometheus Java Simpleclient (from )
+- RabbitMQ Java Client (from http://www.rabbitmq.com)
 - Shaded Guava artifact for use in the DataStax Java driver for Apache Cassandra® (from https://github.com/datastax/java-driver-shaded-guava)
 - config (from https://github.com/typesafehub/config)
-- value (from )
+- graphql-java (from https://github.com/graphql-java/graphql-java)
+- jersey-core-common (from )
+- jersey-core-server (from git://java.net/jersey~code/jersey-server)
+- msal4j (from https://github.com/AzureAD/microsoft-authentication-library-for-java)
 
 ========================================================================
 unknown
 ========================================================================
 The following software have components provided under the terms of this license:
 
+- Checker Qual (from https://checkerframework.org)
+- ClassMate (from http://github.com/cowtowncoder/java-classmate)
+- JSON in Java (from https://github.com/douglascrockford/JSON-java)
 - JUnit Jupiter (Aggregator) (from https://junit.org/junit5/)
-- JUnit Jupiter (Aggregator) (from https://junit.org/junit5/)
-- Mockito (from http://www.mockito.org)
+- Java Native Access (from https://github.com/java-native-access/jna)
+- JavaBeans Activation Framework API jar (from )
+- JavaMail API (from )
+- Objenesis (from http://objenesis.org)
+- RabbitMQ Java Client (from http://www.rabbitmq.com)
+- jakarta.xml.bind-api (from )
 - jersey-container-servlet (from git://java.net/jersey~code/project/jersey-container-servlet)
+- jersey-container-servlet-core (from git://java.net/jersey~code/project/jersey-container-servlet-core)
+- jersey-core-client (from git://java.net/jersey~code/jersey-client)
+- jersey-ext-bean-validation (from )
+- jersey-ext-metainf-services (from )
+- jersey-inject-hk2 (from )
+- jersey-media-jaxb (from )
+- jersey-test-framework-core (from )
+- jersey-test-framework-provider-inmemory (from )
+- jnr-posix (from )
 - org.junit.jupiter:junit-jupiter-api (from http://junit.org/junit5/)
```

Comment:
Some of these are also in "known" sections (e.g. JUnit and `JavaMail API`)

Reply:
Probably in the known sections for their license and down here for the licenses of their dependencies.